Lesson03_Task01.py

```diff
@@ -0,0 +1,75 @@
+"""
+You have list of tuples.
+Each tuple represents:
+# name, age, some sum, last name, sex
+Please do such things:
+# 1 - sort list by age and sex fields
+# 2 - somehow you need to get new list as old list without first two elements and last two elements. Print this new list
+# 3 - in new list calculate total numbers of "female"  and "male" and print it as small table.
+Example:
+# -----------------
+# | sex    | count |
+# -----------------
+# |  female |  11  |
+# |   male  |  23  |
+# -----------------
+advice:
+# female and male calculation can be done vs flat list or you can find your own approach ;)
+"""
+
+people = [
+    ('Alice', 32, 100, 'Johnson', 'female'),
+    ('Bob', 41, 200, 'Smith', 'male'),
+    ('Charlie', 27, 150, 'Jones', 'male'),
+    ('David', 52, 75, 'Williams', 'male'),
+    ('Eve', 18, 300, 'Davis', 'female'),
+    ('Frank', 33, 50, 'Taylor', 'male'),
+    ('Grace', 42, 125, 'Clark', 'female'),
+    ('Henry', 26, 225, 'Lewis', 'male'),
+    ('Ivy', 38, 175, 'Moore', 'female'),
+    ('Jack', 20, 140, 'Harris', 'male'),
+    ('Kate', 37, 110, 'Miller', 'female'),
+    ('Leo', 44, 90, 'Wilson', 'male'),
+    ('Mae', 29, 180, 'Brown', 'female'),
+    ('Nick', 51, 70, 'Davies', 'male'),
+    ('Oliver', 18, 250, 'Collins', 'male'),
+    ('Pete', 36, 160, 'Green', 'male'),
+    ('Quinn', 20, 230, 'Bell', 'female'),
+    ('Remy', 30, 120, 'Foster', 'male'),
+    ('Sara', 28, 140, 'Baker', 'female'),
+    ('Tom', 47, 80, 'Scott', 'male'),
+    ('Ursula', 39, 135, 'Adams', 'female'),
+    ('Vivian', 25, 190, 'Ross', 'female'),
+    ('Wendy', 46, 90, 'Wright', 'female'),
+    ('Xavier', 31, 105, 'Reed', 'male'),
+    ('Yuliana', 22, 200, 'Lopez', 'female'),
+    ('Zack', 48, 60, 'Mitchell', 'male'),
+    ('Adam', 35, 75, 'Davis', 'male'),
+    ('Bella', 27, 125, 'Smith', 'female'),
+    ('Charlie', 44, 115, 'Johnson', 'male'),
+    ('Daisy', 20, 215, 'Miller', 'female'),
+    ('Ethan', 33, 100, 'Taylor', 'male'),
+    ('Fiona', 40, 150, 'Jones', 'female'),
+    ('George', 24, 180, 'Lewis', 'male'),
+    ('Hannah', 22, 200, 'Williams', 'female'),
+    ('Ivan', 29, 160, 'Brown', 'male'),
+    ('Julie', 55, 90, 'Clark', 'female'),
+    ('Kenny', 38, 140, 'Harris', 'male'),
+    ('Luna', 55, 170, 'Smith', 'female'),
+    ('Mike', 55, 55, 'Johnson', 'male')
+]
+
+sorted_people = sorted(people, key=lambda x: (x[1], x[4]))
+new_list = sorted_people[2:-2]
+
+female_count = sum(1 for person in new_list if person[4] == 'female')
```

Comment:
No no no )) Too weird for simple list operations task
Better to flat list and then do count aka flat_people_list.count('female')